interface/patient_file/printed_fee_sheet.php

```diff
@@ -361,20 +360,40 @@ function printlog_before_print() {
 
 // Loop on array of PIDS
 $saved_pages = $pages; //Save calculated page count of a single fee sheet
+$loop_idx = 0; // counter for appt list
+
 
 foreach ($pid_list as $pid) {
+
+    $apptdate = $apptdate_list[$loop_idx]; // parallel array to pid_list
+    $appointment = fetchAppointments( $apptdate, $apptdate, $patient_id=$pid );  // Only expecting one row for pid
```

Comment:
The 4th parameter above looks odd

Reply:
actually, it's the 3rd parameter

Reply:
Thanks Brady for reviewing.  The 3rd param syntax on fetchAppointments call on line 369, is there because I noticed that in the include file (line 359), the 3rd param says $patient_id = null.  I might have been conflating PHP with Python syntax.  But if there is an equal sign for param in function signature, then that means it assigns a default value if param not mentioned in call (in this case null).  It seemed to have worked OK.  I guess I was trying to explicitly state the corresponding param by name, rather than by position.  But in this case the $patient_id does line up position-wise in both func. call and func. signature.  So, go ahead Steve and remove the "$patient_id=" prefix to 3 param.  In other words, just "$pid".  Sorry for any confusion.
https://github.com/openemr/openemr/blob/master/library/appointments.inc.php